aspnetcore/web-api/action-return-types.md

```diff
@@ -10,61 +11,41 @@ uid: web-api/action-return-types
 ---
 # Controller action return types in ASP.NET Core web API
 
-By [Scott Addie](https://github.com/scottaddie)
```

Comment:
Should I also be removing authors as I update topics?

Reply:
Recently, Rick implied that I didn't quite have the right rule in mind or that it had changed. IIRC the original idea was that FTEs/contractors would go away unless ...

* There are community authors on the topic who aren't responsible for most of the text. The FTE/contractor did most of the work, so the byline shouldn't indicate "By {COMMUNITY AUTHOR}" ... it's misleading.
* The FTE/contractor objected to having their name removed for some reason.

I think my removals here are ok, but I can revert later back to them if the team wants to keep them on these. I'll be 👂 for that feedback.